src/providers/twitch/TwitchIrcServer.cpp

```diff
@@ -45,11 +43,10 @@ TwitchIrcServer::TwitchIrcServer()
     , liveChannel(new Channel("/live", Channel::Type::TwitchLive))
     , automodChannel(new Channel("/automod", Channel::Type::TwitchAutomod))
     , watchingChannel(Channel::getEmpty(), Channel::Type::TwitchWatching)
+    , pubsub(new PubSub(TWITCH_PUBSUB_URL))
```

Comment:
This should hopefully not change anything, but if something breaks it might be because of this